Fix(curriculum): external Tower of Hanoi link in CIP (#46434)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/towers-of-hanoi.md

# --description--

> Solve the Towers of Hanoi problem. You have the number of objects as the first parameters, and strings used to identify each of the stacks of objects. create a nested array containing the list of moves, source -> destination.

Reviewer: perhaps:
```suggestion
Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, and followed by the strings used to identify each of the stacks of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

> Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, and followed by the strings used to identify each of the stacks of objects. Create a nested array containing the list of moves, `["source", "destination"]`.

Reviewer 2: ```suggestion
Solve the Towers of Hanoi problem. The number of objects will be given as the first parameter, followed by the strings used to identify each stack of objects. Create a nested array containing the list of moves, `["source", "destination"]`.
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

> For example, object with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move a object from stack `A` to `C`, and the 2nd move was to move a object from stack `B` to `A`.

Reviewer 3: ```suggestion
For example, objects with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move an object from stack `A` to `C`, and the 2nd move was to move an object from stack `B` to `A`.
```

> For example, objects with these parameters `(4, 'A', 'B', 'C')`, will have a nested array `[['A', 'C'], ['B', 'A']]` indicating that the 1st move was to move an object from stack `A` to `C`, and the 2nd move was to move an object from stack `B` to `A`.

Reviewer: ```suggestion
For example, the parameters `(4, 'A', 'B', 'C')`, will result in nested array of moves `[['A', 'C'], ['B', 'A']]`, indicating that the 1st move was to move an object from stack `A` to `C` and the 2nd move was to move an object from stack `B` to `A`.
```

# --instructions--

> Write a function that return the moves to stack the objects in a nested array.

Reviewer 3: ```suggestion
Write a function that returns the moves to stack the objects in a nested array.
```
